server/main.go

```
@@ -44,8 +44,15 @@ func main() {
 	var lis net.Listener
 	var err error
 	if tcpAddr != "" {
+		log.Println("Starting tcp gRPC server")
 		lis, err = net.Listen("tcp", tcpAddr)
 	} else {
+		err = syscall.Unlink(unixPath)
```

I think we could ignore the error when unlink failed 
```go
syscall.Unlink(unixPath)
```